src/lib.rs

```diff
@@ -35,3 +35,6 @@ pub type Key = [u8; KEY_SIZE];
 
 #[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64")))]
 compile_error!("parity-db only supports x86_64 and aarch64");
+
+#[cfg(not(target_endian = "little"))]
+compile_error!("parity-db only supports little-endian platforms");
```

Comment:
Make sense (keeping support would require a CI test indeed).

Reply:
Yeah, it is not tested properly. The platform restriction of x86_64 and aarch64 limits it to litle endian anyway.